semgrep-core/src/fixing/Autofix.ml

```diff
@@ -0,0 +1,70 @@
+(* Nat Mote
+ *
+ * Copyright (C) 2019-2022 r2c
```

Comment:
I would put just 2022 but as you want.

Reply:
I copied the existing license headers that I found and updated 2021 to 2022. I think we should have some consistent license header along with automated enforcement but that's out of scope for this PR.